pipeline.py

```diff
@@ -69,31 +93,123 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+
+        for article in articles:
+            raw_text = article.get_raw_text()
+            processed_tokens = self._process(raw_text)
+
+            cleaned_tokens = []
+            single_tagged_tokens = []
+            multiple_tagged_tokens = []
+
+            for processed_token in processed_tokens:
+                cleaned_tokens.append(processed_token.get_cleaned())
+                single_tagged_tokens.append(processed_token.get_single_tagged())
+                multiple_tagged_tokens.append(processed_token.get_multiple_tagged())
+
+            article.save_as(' '.join(cleaned_tokens), 'cleaned')
+            article.save_as(' '.join(single_tagged_tokens), 'single_tagged')
+            article.save_as(' '.join(multiple_tagged_tokens), 'multiple_tagged')
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        # Gets rid of '-\n's that break up words, so we won't get 'кот-' 'орый' as example (Should be: 'который')
+        text = raw_text.replace('-\n', '')
+
+        # [:500] is there because scientific articles are huge and CI takes too long instead (way more than 5 mins)
+        result = Mystem().analyze(text[:500])
```

Comment:
and what? do you suggest collecting shortened dataset? that is not acceptable. 

1. think of ways of making it faster
2. does it take sol long on your computer as well?
3. ensure you finally analyze the whole text
4. communicate such a question (only if you fail to resolve it yourself) to a forum for mentors to help. 

When properly configured, MyStem is super quick analyzer even for large texts